control-plane-operator/controllers/hostedcontrolplane/configoperator/reconcile.go

```diff
@@ -150,6 +151,13 @@ func ReconcileDeployment(deployment *appsv1.Deployment, image, hcpName, openShif
 			},
 		},
 	}
+
+	if explicitNonRootSecurityContext {
+		deployment.Spec.Template.Spec.SecurityContext = &corev1.PodSecurityContext{
```

Comment:
I wasn't aware this can be set for all containers in the pod, if we can do that setting it explicitly for each container as we now do in the context of DeploymentConfig doesn't seem to be needed?

Reply:
Setting the Security Context at the pod spec level for all the components now. Makes it easier for us.